ratel/src/parser/expression.rs

```diff
@@ -141,6 +141,25 @@ create_handlers! {
         par.alloc_at_loc(start, end, expression)
     };
 
+    pub const NEW = |par| {
+        let (start, op_end) = par.lexer.loc();
+        let op = OperatorKind::from_token(par.lexer.token).expect("Must be a prefix operator");
```

Comment:
The check and the variable are unnecessary, you are guaranteed that the token is `NewOperator` now :)